src/server.h

```diff
@@ -609,6 +609,16 @@ typedef enum {
 #define PAUSE_ACTION_EVICT            (1<<3)
 #define PAUSE_ACTION_REPLICA          (1<<4) /* pause replica traffic */
 
+/* Sets log format */
+#define LOG_FORMAT_DEFAULT 0
+#define LOG_FORMAT_LOGFMT 1
```

Comment:
Would prefer to make these a typedef enum. Although some of the other configs don't do this, the benefit is that when using a case switch statement, we get a warning if we are missing one of the enum values.